contribs/gnodev/pkg/dev/node.go

```diff
@@ -497,6 +503,15 @@ func (n *Node) rebuildNodeFromState(ctx context.Context) error {
 
 	// Load genesis packages
 	pkgs, err := n.loader.Load(n.paths...)
+	filterPackages := make([]packages.Package, 0, len(pkgs))
+	for _, pkg := range pkgs {
+		if !cachepath.Get(pkg.Path) {
+			filterPackages = append(filterPackages, pkg)
+		} else {
+			n.logger.Error("Can't reload package, package conflict in ", "path", pkg.Path)
+		}
+	}
+
```

Comment:
no need to allocate another array here, you should filter this directly in `generateTx`